Testing/Exec/TranEval/GPU_misc.H

```diff
@@ -45,7 +45,7 @@ initialize_data(int i, int j, int k,
 	rho(i,j,k) = 0.01 + dRho * std::sin(2.0*pi*y/P[1]);
 
 	amrex::Real dummy = 0.0;
-	for (int n = 0; n < NUM_SPECIES; n++) {
+	for (int n = 0; n < NUM_SPECIES-1; n++) {
```

Comment:
What if the last species doesn't have enough mass fraction so that this version ends up putting a negative mass fract there?

Reply:
It is not possible: if you look at how each mf[n] is constructed they are bounded between Y_hi[n] and Y_lo[n]. The sum of Y_lo[n] (for 1: NUM_SPEC-1) is 0 and of Y_hi[n] is (1- 1.0 / NUM_SPECIES ). So the sum will always be smaller than 1 and we're good.